packages/frontend/src/views/create-event.vue

```diff
@@ -209,6 +227,12 @@ export default defineComponent({
       selectedTimes.value.splice(index, 1)
     }
 
+    const result = ref('')
+
+    const updateMarkdown = debounce((e:Event): void => {
```

Comment:
this is just gross

it should be 

```ts
const updateMarkdown = debounce((e: Event): void => {
  result.value = markdown.render((e.target as HTMLTextAreaElement).innerText)
}, 500)
```

Also you should change from `HTMLTextAreaElement` to `HTMLSpanElement` ([reference](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSpanElement))